.eslintrc.js

```diff
@@ -2,17 +2,23 @@ module.exports = {
   root: true,
   env: {
     "cypress/globals": true,
-    browser: true,
-    node: true
+    node: true,
+    browser: true
   },
   extends: [
-    'standard',
-    'plugin:vue/essential'
+    'plugin:vue/recommended',
+    '@vue/standard',
+    'plugin:vue-a11y/base'
   ],
   plugins: [
-    "cypress"
+    'cypress',
+    'vue-a11y'
```

Comment:
Want to replace this with the new linter?

Reply:
Yes, well-remembered!

Reply:
Do you want to update?